pkg/install/install.go

```diff
@@ -200,7 +205,7 @@ func Install(ctx context.Context, image string, opts *Options) error {
 	s.Success()
 
 	s = opts.Progress.New(fmt.Sprintf("Installing APIServer and Controller (image %s)", image))
-	if err := applyDeployments(ctx, image, *opts.APIServerReplicas, *opts.ControllerReplicas, apply, c); err != nil {
+	if err := applyDeployments(ctx, image, *opts.APIServerReplicas, *opts.ControllerReplicas, *opts.UseCustomCABundle, apply, c); err != nil {
```

Comment:
Any particular reason to not use `opts.Config.UseCustomCABundle` here? Same question applies to everything below.

Reply:
The main problem with this is that there is an option in install called `install -o yaml` which print out the deployment object in yaml without saving anything into the config. So in here if you reference anything in config it will be nil. So we use the static flag here. We still save the flag into the config so that builder deployment reference the config and add the volumn mount if needed.

Reply:
Actually @tylerslaton this is a good point. I changed it so it can intialize the config when you run `install -o yaml` so we just use the flag in the config.